src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is being used here which can impact performance. It's better to set box-sizing specifically for elements that require it.